boltdesignsystem/bolt
# DS-647 Academy Doc Pages Move

## Jira

https://pegadigitalit.atlassian.net/browse/DS-647

## Summary

Academy docs pages were moved

## Details

- `Academy` was moved from The `Pages` folder to the `Tests` folder.
- Twig namespace paths in `docs-site/.boltrc.js` were updated
- Import in the `doc-site/src/index` was updated

## How to test

Pull the branch make sure docs were moved properly and there are no regressions.
